book/scheduling.md

```diff
@@ -393,15 +387,16 @@ Chapter 10 is not very useful and a huge performance footgun. Some
 browsers are now moving to deprecate synchronous `XMLHttpRequest`.]
 Python's `Thread` class lets us do better:
 
-    threading.Thread(target=callback).start()
+``` {.python .example}
+threading.Thread(target=callback).start()
+```
     
-This code creates a new thread and then immediately returns. The
-`callback` then runs in parallel, on the new thread, while the initial
-thread continues to execute later code.
+This code creates a new thread and then continues to execute. The
```

Comment:
I think the old text was better. "continues to execute" is ambiguous

Reply:
I guess my real objection is “later code” which is kind of vague. But not critical in any case.

Reply:
Is it vague? Doesn't seem to to me..

Reply:
If you'd rather revert it that's fine.

Reply:
I'll do it.